index.html

```diff
@@ -857,7 +857,7 @@ <h4>blockchainAccountId</h4>
                 <a href="https://w3id.org/security#blockchainAccountId">security-vocab</a>
               </td>
               <td>
-                <a href="https://www.w3.org/ns/did/v1">DID Core</a>
+                <a href="https://w3id.org/security/suites/secp256k1-2020/v1">https://w3id.org/security/suites/secp256k1-2020</a>
```

Comment:
I think the [definition](https://w3c-ccg.github.io/security-vocab/#blockchainAccountId) in the security vocab would work here?  Other properties here point there already.

```suggestion
                <a href="https://w3id.org/security/v3-unstable">security-vocab-v3-unstable</a>
```

Reply:
@msporny @tplooker do you all have a preference for avoiding sec-v3 vs using suite specific contexts? My understanding was that we probably won't ever see sec-v3 define these kinds of terms when it stabilizes.

Reply:
> do you all have a preference for avoiding sec-v3 vs using suite specific contexts?

Digital Bazaar's implementation experience is driving us very strongly towards suite-specific contexts. We have now implemented that pattern across all our issuer and verifier infrastructure as well as Veres Delta and Veres One. It worked well in each of those scenarios, the Veres One implementation being quite different from the first two (utilizing zcaps as a separate context in addition to the cryptosuite contexts).

The theory when suite-specific contexts were first proposed is that they would lead to better composability; we can now confirm that is true for the use cases we've applied them to. This might not be a surprise to those that use `#include`/`require`/`import` in programming languages. Developers have always tried to make the externally defined things they use composable... suite-specific contexts is the semantic version of `#include`.

The only downsides seem to stem from not being able to do additive JSON-LD scoped and protected classes (but to be clear, sec-v3 wouldn't have solved that issue either) AND not being able to do a simple context value check when operating in JSON-only mode.

We would probably be ambivalent OR argue against sec-v3 at this point in time. Monolithic contexts make it harder to keep pace with industry needs and innovate.

We would be very supportive of suite-specific (Ed25519-2020) and use-specific (ZCAP) contexts.